src/middlewared/middlewared/plugins/interface/netif_linux/ethernet_settings.py

```diff
@@ -72,6 +72,19 @@ def __mediainfo__(self):
             # of media info (ethtool binary didnt report anything either)
             # so ignore these errors
             pass
+        except NoSuchDevice:
```

Comment:
Do you think it's better to do this on L69 perhaps like `(NotSupportedError, NoSuchDevice)` ? This way we can ignore specific exceptions and not have their own `pass` / block for them ?